iris_grib/__init__.py

```diff
@@ -195,6 +178,12 @@ def __init__(self, grib_message, grib_fh=None, auto_regularise=True):
         warn_deprecated('Deprecated at version 1.10')
```

Comment:
@marqh I was planning to do a complete sweep through the code to address any outstanding deprecation warnings, once the backlog of main PRs are merged.